membership-attribute-service/app/components/TouchpointComponents.scala

```diff
@@ -148,7 +146,7 @@ class TouchpointComponents(
   lazy val futureCatalog: Future[CatalogMap] = catalogService.catalog
     .map(_.fold[CatalogMap](error => { println(s"error: ${error.list.toList.mkString}"); Map() }, _.map))
     .recover { case error =>
-      logError(scrub"Failed to load the product catalog from Zuora due to: $error")
+      logger.error(scrub"Failed to load the product catalog from Zuora due to: $error")
```

Comment:
this is a more typical syntax for log statements, it's the same as the standard scala-logging way (apart from we "scrub" to avoid sending PII to sentry